src/game.cpp

```diff
@@ -5758,33 +5753,21 @@ int main(int argc, char** argv)
 
 				doFrames();
 
-				if ( !gamePaused )
+				if ( /*newui*/ 0 ) 
```

Comment:
umm...

Reply:
Is there a better way to do this? I had to do a double take when I read that line the first time

Reply:
I've removed it in my branch, deprecated anyway should always just be ingameHud()

Reply:
Ah ok good to know, thanks (haven't had a chance to look over your branch yet)